packages/app/src/app/pages/Dashboard/Content/CreateNewSandbox/NewSandboxModal/Imports/StackbitButton.tsx

```diff
@@ -32,13 +32,18 @@ function openStackbit(username: string) {
   );
 }
 
-interface Props {
+interface IStackbitButtonProps {
   username: string;
-  style?: React.CSSProperties;
 }
 
-export const StackbitButton = ({ username, style }: Props) => (
-  <Button style={style} small onClick={() => openStackbit(username)}>
+export const StackbitButton: React.FC<IStackbitButtonProps> = ({
+  username,
+}) => (
+  <Button
+    css="margin-top: 1rem; float: right;"
```

Comment:
I don't think this works, what is this css?

Reply:
the CSS prop is part of styled components, the babel plugin reads that prop and generates classnames, it's an alternative to using `styled`. Otherwise you have to include `style` in your component interface, if you don't typescript yells at you. This is the preferred way to do inline styles, though we should avoid it as much as possible.

Reply:
for reference:

https://www.styled-components.com/docs/api#css-prop